src/btree/bt_debug.c

```diff
@@ -475,23 +475,20 @@ __debug_hs_cursor(WT_DBG *ds, WT_CURSOR *hs_cursor)
 
     switch (hs_upd_type) {
     case WT_UPDATE_MODIFY:
-        WT_RET(ds->f(ds,
-          "\t"
-          "hs-modify: %s\n",
-          __wt_time_window_to_string(&cbt->upd_value->tw, time_string)));
+        WT_RET(ds->f(ds, "\t%s\n", __wt_time_window_to_string(&cbt->upd_value->tw, time_string)));
         if (F_ISSET(ds, WT_DEBUG_UNREDACT_ALL)) {
-            WT_RET(ds->f(ds, "\tV "));
+            WT_RET(ds->f(ds, "\ths_modify: "));
             WT_RET(__debug_modify(ds, ds->hs_value->data));
             WT_RET(ds->f(ds, "\n"));
         } else
-            WT_RET(ds->f(ds, "\tV {REDACTED}\n"));
+            WT_RET(ds->f(ds, "\ths_modify: {REDACTED}\n"));
```

Comment:
While I think it's fine to have the `\t` with and the strings as is (i.e. `"\ths_modify`) - should we stay consistent with [this style of formatting](https://github.com/wiredtiger/wiredtiger/pull/10145/files#diff-eec07b0b03dc98e7eba3b478e28ca13a058753c06c4643ee4595f4887b83e609L491) or do you reckon it's fine to leave it as is (considering a change like this will include adding a lot of new lines since we've added many `\t`)?